### PR TITLE
Symspell

### PR DESCRIPTION
**Pull request by AlvaroCavalcante**
## Symspell Update!

- Added an optional parameter to call Symspell 
- When the parameter is True, Symspell is loaded to memory and the output is correct based on suggestions!

### CODE REVIEW

File: src/auxiliary.py [hunk 1]
```diff
@@ -378,3 +379,20 @@ def get_image_from_url(url):
...
+
+def load_dict_to_memory():
+    sym_spell = SymSpell(max_dictionary_edit_distance=2, prefix_length=7)
+    sym_spell.load_pickle('./src/dictionary/dictionary.pkl')
+    return sym_spell
+
+def get_word_suggestion(symspell, input_term):
+    get_digits = re.findall('\d+', input_term)
+
+    if len(get_digits) == 0:
```
Comment (Lucs1590) [inline src/auxiliary.py:391]: Is it correct `== 0` here?

**Review by Lucs1590 — CHANGES_REQUESTED** [review]
Your code is very good, my friend, but the integration tests for load_dict_to_memory and get_word_suggestion were missing. Would it be possible to insert them?

File: src/auxiliary.py (hunk 1 above)
Comment (AlvaroCavalcante) [reply src/auxiliary.py:391]: Ok, I'll add the tests then.

File: src/ocr_table.py [hunk 2]
```diff
@@ -13,28 +13,37 @@ class OcrTable:
...
         started_time = time()
 
         input_type = aux.get_input_type(image)
         self.text = process_image(image, input_type)
 
+        if self.spell_corrector:
+            sym_spell = aux.load_dict_to_memory()
+            self.text = [aux.get_word_suggestion(
+                sym_spell, input_term) for input_term in self.text.split(' ')]
+            self.text = ' '.join(self.text)
```
Comment (Lucs1590) [inline src/ocr_table.py:28]: @AlvaroCavalcante, please put this inside the `run_pipeline` function?
https://github.com/Lucs1590/Nkocr/blob/c59128d4415e6e2e100771316c9fc418d9837caf/src/ocr_table.py#L70

**Review by Lucs1590 — CHANGES_REQUESTED** [review]

**Review by Lucs1590 — APPROVED** [review]

**Pull request merged**